content/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -154,8 +154,38 @@ Any new feature flags introduced since the previous OpenZFS version that was int
 
 For more details on feature flags, see [OpenZFS Feature Flags](https://openzfs.github.io/openzfs-docs/Basic%20Concepts/Feature%20Flags.html) and [OpenZFS zpool-feature.7](https://openzfs.github.io/openzfs-docs/man/7/zpool-features.7.html).
 
+## 24.10.1 Changelog
+
+**December 12, 2024**
+
+iXsystems is pleased to release TrueNAS 24.10.1!
+This is a maintenance release and includes refinement and fixes for issues discovered after the 24.10.0 and 24.10.0.X releases.
+
+* Prevent incorrect translation of LDAP basedn to kerberos realm ([NAS-132192](https://ixsystems.atlassian.net/browse/NAS-132192)).
```

Comment:
Assuming "basedn" was taken from the referenced ticket, but we use "Base DN" in other parts of the docs. 
https://www.truenas.com/docs/scale/24.10/scaleuireference/credentials/directoryservices/ldap/